base/ref.go

```diff
@@ -26,6 +30,9 @@ func SetPublishStatus(r repo.Repo, ref *repo.DatasetRef, published bool) error {
 		return fmt.Errorf("can't publish datsets that are not in your namespace")
 	}
 
+<<<<<<< HEAD
```

Comment:
Merge conflicts need to be resolved in this commit.

Reply:
yeah I made a bunch of garbage errors on a rebase, force-pushed the last commit